app/controllers/users_controller.rb

```diff
@@ -1,4 +1,13 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
quá nửa actions( tổng: 7 action theo chuẩn resful) -> tìm hiểu về except thay vì only nhé
```suggestion
  before_action :logged_in_user, except: %i(new create show)
```